launchable/test_runners/nunit.py

```diff
@@ -162,9 +166,21 @@ def record_tests(client, report_xml):
 """
 
 
-def _replace_fixture_to_suite(paths) -> List[Dict[str, str]]:
+def _replace_fixture_to_suite(tp_str: str) -> List[Dict[str, str]]:
```

Comment:
What is tp? test_path?

Reply:
[be00068](https://github.com/launchableinc/cli/pull/641/commits/be000687bc284ed8f7440dc18855c9b6e0d39163)